src/plotman/log_parser.py

```diff
@@ -0,0 +1,140 @@
+import re
+from plotman.plotinfo import PlotInfo
+
+class PlotLogParser:
+    """Parser for a finished plotting job"""
+
+    def parse(self, filename: str) -> PlotInfo:
```

Comment:
How about taking an open file object?  Makes this more flexible and encourages explicit IO control at the top level.